lib/__tests__/tickplate.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-template-curly-in-string */
```

Comment:
Should I change it?
By default, Jest uses this [regex](https://facebook.github.io/jest/docs/en/configuration.html#testregex-string) to detect test files.